docs/README-migration.md

```diff
@@ -1,1318 +1,1322 @@
-# Migrating to SDL 3.0

```

Comment:
You accidentally changed the eol's of this file.

Reply:
Oh I see, it has CRLF on the main branch, let me adjust.

Reply:
Perhaps it needs a python fix, similar to the [one applied here](https://github.com/libsdl-org/SDL/commit/796713b9d54d1e93b991d4b5e5365033a0da7ba6)?

Reply:
Done.